transmit-lib/src/cmr/transmit/search.clj

```diff
@@ -89,6 +89,28 @@
       (errors/internal-error!
         (format "Granule search failed. status: %s body: %s" status body)))))
 
+(defn-timed find-granule-references-return-errors
+  "Find granules by parameters in a post request. If the search fails, returns the failure
+  status and body."
+  [context params]
+  (let [conn (config/context->app-connection context :search)
+        request-url (str (conn/root-url conn) "/granules.json")
+        request-body (-> params
+                         (dissoc :token)
+                         (assoc :page_size 0))
+        token (:token params)
+        header (ch/context->http-headers context)
+        response (client/post request-url
+                              (merge
+                                (config/conn-params conn)
+                                {:body (codec/form-encode request-body)
+                                 :content-type mt/form-url-encoded
+                                 :throw-exceptions false
+                                 :headers (if token (assoc header config/token-header token) header)}))
+        {:keys [status body]} response]
+    (when-not (= status 200)
+      response)))
```

Comment:
change `response` to `body`.

Reply:
I was thinking that since we don't know what exactly the status code will be (only that it isn't 200) it might be useful to have the whole response, but since I'm not looking for it at all in the validation code, I can't argue with this change.